### ISSUE TITLE
CkanCatalogGroup should handle filterQuery correctly

### ISSUE THREAD

**Issue by mwu2018** [TerriaJS/terriajs#5331]
The current implementation assumes that each element in filterQuery array of CkanCatalogGroup is an Object that has key-value pair such as
```
{
  "fq": "(organization:alpine-shire-council AND res_format:(geojson OR GeoJSON OR kml OR KML OR kmz OR KMZ OR wms OR WMS OR CSV-GEO-AU OR csv-geo-au OR aus-geo-csv OR \"Esri REST\"))"
}
```

However, an array element can also be a pure query string such as 
```
  "fq=+(res_format%3Awms%20OR%20res_format%3AWMS)"
```
The code should also handle that case.

**Comment by mwu2018** [comment]
Changed to medium priority as this will not impact vic dt anymore. However, it is required for v8 national map.

**Comment by nf-s** [comment]
Closed by #5332
